app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -279,10 +291,56 @@ public int onStartCommand(Intent intent, int flags, int startId) {
             createIVButton();
             createArcPointer();
             createArcAdjuster();
+            /* Assumes MainActivity initialized ScreenGrabber before starting this service. */
```

Comment:
This is one of the temporary hacks I was referring to in the PR text. It'll go away.